pkg/webui/components/page-title/index.js

```diff
@@ -30,7 +30,7 @@ const PageTitle = ({ title, values, tall, className, hideHeading, children }) =>
   return hideHeading ? (
     pageTitle
   ) : (
-    <Row>
+    <Row className={className}>
```

Comment:
Do you still need this? Cannot find any usage of `PageTitle` with `className` prop